.eslintrc.json

```diff
@@ -30,10 +27,42 @@
     "testing-library/prefer-explicit-assert": "warn",
     "testing-library/prefer-presence-queries": "error",
     "testing-library/prefer-screen-queries": "error",
-    "testing-library/prefer-wait-for": "error",
-    "no-use-before-define": "off",
-    "@typescript-eslint/no-use-before-define": [
-      "error"
-    ]
-  }
+    "testing-library/prefer-wait-for": "error"
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts", "**/*.tsx"],
+      "env": {"browser": true, "es6": true, "node": true},
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+      ],
+      "globals": {"Atomics": "readonly", "SharedArrayBuffer": "readonly"},
```

Comment:
I'm not familiar with these globals? What are they or can we delete them and everything still works?